es67x/src/main/scala/tech/beshu/ror/es/IndexLevelActionFilter.scala

```diff
@@ -92,11 +92,16 @@ class IndexLevelActionFilter(nodeName: String,
     startRorInstance()
   }
 
-  private def auditSinkCreator: AuditSinkCreator = {
-    case AuditCluster.LocalAuditCluster =>
-      new EsAuditSinkService(client)
-    case remote: AuditCluster.RemoteAuditCluster =>
-      HighLevelClientAuditSinkService.create(remote)
+  private def auditSinkCreator: AuditSinkCreator = new AuditSinkCreator {
```

Comment:
Maybe instead of throwing the illegal state ex, we can make this state impossible?
Eg. by having two AuditSinkCreator traits? One supporting only index sink. And the second one supporting both?